src/routes/Routes.tsx

```diff
@@ -2,15 +2,21 @@ import React, {
   // useEffect,
   useMemo,
 } from 'react';
-import { Route, Routes as RoutesSwitch, Navigate } from 'react-router-dom';
+import {
+  Route,
+  Routes as RoutesSwitch,
```

Comment:
I'm pretty sure this was changed by Raul and the reason behind renaming it is to avoid conflict with the component itself which is also called `Routes`

Reply:
Correct. Exactly that.

You could rename it if you feel like it, I don't really care what name it ends up with